### DISCUSSION TITLE
Support for NIST SP 800-161 Appendix F, for Executive Order 14028?

### DISCUSSION THREAD

**Discussion by rjb4standards** [usnistgov/OSCAL#1098] [Q&A] [answered]
Are there any plans to produce OSCAL artifacts for NIST 800-161 Appendix F, supporting Executive Order 14028?

**Comment by iMichaela** [comment]
@rjb4standards - Unfortunately NIST OSCAL team does not have the cycle, any time soon, to generate the requested information.
If you are interested in the F-x tables in the Appendix F of SP 800-161, those are good candidates for OSCAL profile(s). There are 2 ways of including the E.O 14028 mapping: 
1) The catalog of controls will have to be first altered to include the E.O.14028 mapping for the listed controls by using `props`, then the profile will import only the controls listed in the tables, bringing forward the mapping in the process. 
2) Alternatively, one can generate first a resolved profile with the listed controls from the F-x tables and then alter the obtained catalog to add the E.O 14028 using again `props`. I think I prefer 2), so the 800-53 catalog remains intact.

**Comment by rjb4standards** [comment]
Thank you for the quick response, Michaela.

**Comment by sunstonesecure-robert** [comment]
I can bring it up to the CNCF supply chain group. Also The Kubernetes
Policy WG is working on Profile alignment, inclusive of supply chain
management policies, so I can discuss if we can contribute back a PR to
OSCAL.  No guarantees but we'll discuss options.

**Comment by rjb4standards** [comment]
Thanks for offering to research this matter. I know that NIST SCRM is working on update to Appendix F to provide guidance needed to implement EO 14028 and the Senate passed legislation on Wednesday that would also fit into this category, i.e., training regarding cybersecurity supply chain risk assessments.

**Comment by aj-stein-nist** [comment]
> I can bring it up to the CNCF supply chain group. Also The Kubernetes Policy WG is working on Profile alignment, inclusive of supply chain management policies, so I can discuss if we can contribute back a PR to OSCAL. No guarantees but we'll discuss options.

Do you have more details on this? You are looking into 800-161 (either appendix) and contributing it back, @sunstonesecure-robert?

**Comment by aj-stein-nist** [comment]
FYI we also have an open call in usnistgov/OSCAL#1080 for use cases in components/SSPs. Not 100% relevant to this (re catalog, profile creation, and/or catalog mappings), but I am not sure someone has asked about "a component in a def or SSP that shows supporting evidence around 800-161 controls" yet. Just a thought! :-)

**Comment by rjb4standards** [comment]
Supporting evidence will be key for SP 800-161, but SBOM vulnerability reporting is also gaining momentum. This week Cyclone DX announced V 1.4 with support for vulnerability reporting in their NTIA supported SBOM standar. I posted an article recently about the various methods for vulnerability reporting after receiving a message from Allan Friedman of CISA: https://energycentral.com/c/um/terminology-confusion-regarding-vulnerability-reporting

**Comment by aj-stein-nist** [comment]
> Supporting evidence will be key for SP 800-161, but SBOM vulnerability reporting is also gaining momentum. This week Cyclone DX announced V 1.4 with support for vulnerability reporting in their NTIA supported SBOM standar. I posted an article recently about the various methods for vulnerability reporting after receiving a message from Allan Friedman of CISA: https://energycentral.com/c/um/terminology-confusion-regarding-vulnerability-reporting

I will have to look later since this requires a membership and I cannot access it even with Javascript disabled. I will have to review from a personal workstation another time.

If you see how evidence of 800-161 and/or SBOM vulnerability reporting would fit into documenting an information system for federal government use and would like to experiment with that, I would really appreciate a quick summary of the specific use case in usnistgov/OSCAL#1080. Eventually the team will vet those requests and try to design examples after we hear from the community are of the most interest.

Michaela spoke to the catalog thing, and I agree with that! It is a tall order that would need community effort. :-)

**Comment by rjb4standards** [comment]
Thanks, Alexander. The next release of SP 800-161 is expected to come out sometime in February. Then we'll all have a better sense for what those Appendix F EO 14028 requirements will be, then we should be able to construct a use case for EO 14028.

**Comment by iMichaela** [comment]
@aj-stein-nist - we can take it internally at NIST with the authors. If Jon et. all. wants to release the data in OSCAL, and **if we can get a little help,** we might be able to generate **the SP 800-161 Appendix F tables** with the mapping to the EO 14028 in OSCAL. Examples, use cases, and SBOM beyond the existing `inventory` is a lot more of work. As of today, we have a full plate with the conference, the workshop and getting the updated for the OSCAL 1.1.0 out.

@rjb4standards - BUT if the community can pitch in and become the driving force for this effort, we are here to provide guidance starting today. The lowest hanging fruit would be the Appendix F, F-x tables as profiles with the controls enhanced to capture the mapping to the EO requirements. **I believe that the OSCAL 1.1.0's Mapping Model could also be used if waiting for the OSCAL 1.1.0 release is acceptable.**

**Comment by rjb4standards** [comment]
Thanks, Michaela.
Reliable Energy Analytics (REA) stands ready to work with any entity in the EO 14028 community that wants to start working/planning their implementation today following NIST SP 800-161 Appendix F and the NTIA SBOM requirements, including the pursuit of an OSCAL artifact, if the EO 14028 community wishes to pursue this effort.

**Comment by iMichaela** [comment]
@rjb4standards - This is great. We can provide guidance to community members interested in rolling sleaves.

**Reply by GaryGapinski** [reply]
Can NIST supply SP 800-161 in a format other than PDF? (I presume there is at least one precursor format).

**Comment by david-waltermire** [comment, marked as answer]
usnistgov/oscal-content#22 is an issue for doing this. As mentioned in this thread, we do not have the cycles to work on this at the moment and there is now SP 800-53 rev5 [OSCAL content](https://github.com/usnistgov/oscal-content/tree/main/nist.gov/SP800-53) which contains the privacy-related controls.
